Sources/UIDeviceExtensions.swift

```diff
@@ -85,6 +85,18 @@ extension UIDevice {
         return UIDevice.current.userInterfaceIdiom == UIUserInterfaceIdiom.pad
     }
 
+    /// EZSE: Returns path of documents directory
+    class func documentsDirectoryPath() -> String {
```

Comment:
I dont understand how this is a UIDeviceExtension? There is no reference to self or any static references to UIDevice anywhere here.